app/org/sagebionetworks/bridge/stormpath/StormpathAccountDao.java

```diff
@@ -278,7 +278,27 @@ public Account getAccount(Study study, String identifier) {
         checkNotNull(study);
```

Comment:
In this class, mostly moving methods around to cleanup, not really deleting or adding.